virttest/remote_commander/remote_runner.py

```diff
@@ -26,6 +26,13 @@
 import messenger as ms
 
 
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+handler = logging.FileHandler("/tmp/remote_runner.log", "w", encoding=None,
```

Comment:
How about get temp log file like below:
tempfile.mktemp(suffix='.log', prefix='remote_runner', dir=data_dir.get_tmp_dir()),
so that your log will not be overwrite.